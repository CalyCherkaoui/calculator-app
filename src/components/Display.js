import React from 'react';
import PropTypes from 'prop-types';

const Display = ({ result }) => (
  <div className="displayer">
    {result}
  </div>
);

Display.defaultProps = { result: '0' };
Display.propTypes = PropTypes.string.isRequired;

export default Display;
