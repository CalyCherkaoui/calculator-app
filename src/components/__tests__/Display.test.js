import React from 'react';
import renderer from 'react-test-renderer';
import Display from '../Display';

describe(
  'Display component',
  () => {
    it('Matches Display snapshot', () => {
      const comp = renderer.create(<Display />);
      expect(comp).toMatchSnapshot();
    });
  },
);
