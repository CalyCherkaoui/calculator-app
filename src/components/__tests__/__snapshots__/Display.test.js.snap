// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display component Matches Display snapshot 1`] = `
<div
  className="displayer"
>
  0
</div>
`;
